RecursiveExtractor/Extractors/RarExtractor.cs

```diff
@@ -89,10 +122,43 @@ public IEnumerable<FileEntry> Extract(FileEntry fileEntry, ExtractorOptions opti
             {
                 Logger.Debug(Extractor.DEBUG_STRING, ArchiveFileType.RAR, fileEntry.FullPath, string.Empty, e.GetType());
             }
-
+            var needsPassword = false;
```

Comment:
Not a big problem, but it would be nice to not have the code duplication between Extract and ExtractAsync. Could be a future refactor, too.

Reply:
Done.